nx-X11/programs/Xserver/dix/property.c

```diff
@@ -531,6 +531,7 @@ ProcGetProperty(ClientPtr client)
 	pProp = pProp->next;
     }
 
+    memset(&reply, 0, sizeof(xGetPropertyReply));
```

Comment:
I think memset is only needed where a reply structure is reused. Upstream did not add it everywhere. Ports to NXproperty.c should be done of course. I can so that at the end of the month.

----- Ursprüngliche Nachricht -----
Von: "Mihai Moldovan" <notifications@github.com>
Gesendet: ‎15.‎06.‎2015 02:53
An: "ArcticaProject/nx-libs" <nx-libs@noreply.github.com>
Cc: "uli42" <uli42@gmx.de>
Betreff: Re: [nx-libs] Pr/backported fixes (#50)

In nx-X11/programs/Xserver/dix/property.c:
> @@ -531,6 +531,7 @@ ProcGetProperty(ClientPtr client)
>  	pProp = pProp->next;
>      }
>  
> +    memset(&reply, 0, sizeof(xGetPropertyReply));
Needs to be backported to nx-X11/programs/Xserver/hw/nxagent/NXproperty.c. Also, there are many other places in NXproperty.c where events are generated not correctly zero-initialized... maybe do them, too?
—
Reply to this email directly or view it on GitHub.